NetProbe/Utility/TestStorage.mm

```diff
@@ -0,0 +1,120 @@
+//
+//  TestStorage.m
+//  NetProbe
+//
+//  Created by Lorenzo Primiterra on 11/06/16.
+//  Copyright © 2016 Simone Basso. All rights reserved.
+//
+
+#import "TestStorage.h"
+
+@implementation TestStorage
+
++ (NSArray*)get_tests{
```

Comment:
Shouldn't this function call `[self checkTests]` such that we always have an array to return? Or, is there a case where it makes sense to return `nil` to signal something to the caller?